lib/profile/commands/configure.rb

```diff
@@ -105,6 +105,7 @@ def display_details
       def save_answers(answers)
         Config.data.set(:cluster_type, value: cluster_type.id)
         Config.save_data
+        Config.set_permission
         cluster_type.save_answers(answers)
```

Comment:
This line (`cluster_type.save_answers`) is the part we want to enforce the permissions for, not the `Config.save_data` method above it.